src/main/java/com/example/homework10/Mapper/MusicMapper.java

```diff
@@ -11,6 +11,9 @@ public interface MusicMapper {
     @Select("select * from music")
     List<Music> findAll();
 
+    @Select("select * from music where id = #{id}")
```

Comment:
SQLの予約語は大文字にしましょうか。
このPRでやらなくてもよいのでfindAllやcreateMusicなど他のメソッドも横並びで直しましょう。
```suggestion
    @Select("SELECT * FROM music WHERE id = #{id}")
```

Reply:
修正させて頂きました。